diff --git a/node_modules/react-native-video/android/build.gradle b/node_modules/react-native-video/android/build.gradle
index 922bd9e..297ba9b 100644
--- a/node_modules/react-native-video/android/build.gradle
+++ b/node_modules/react-native-video/android/build.gradle
@@ -17,6 +17,6 @@ android {
 
 dependencies {
     //noinspection GradleDynamicVersion
-    provided 'com.facebook.react:react-native:+'
-    compile 'com.yqritc:android-scalablevideoview:1.0.4'
+    compileOnly 'com.facebook.react:react-native:+'
+    implementation 'com.github.MatrixFrog:android-scalablevideoview:v1.0.4-jitpack'
 }
